src/tests/test_common/raft_repl_test_base.hpp

```diff
@@ -0,0 +1,629 @@
+/*********************************************************************************
```

Comment:
Moved common code from test_raft_repl_dev.cpp to raft_test_base.hpp. No change in code. Just moved the classes.